playwright-ct.config.ts

```diff
@@ -0,0 +1,43 @@
+import { defineConfig, devices } from '@playwright/experimental-ct-react17';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
```

Comment:
Did we already ask Gavin about including this other version of playwright? Or are we planning to make a case to him on why it makes testing better?